app/recipe/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+
+from .models import Recipe, Ingredient
+
+admin.site.register(Recipe)
+admin.site.register(Ingredient)
```

Comment:
This is good, but if you want to make it better, look at the [Inline](https://docs.djangoproject.com/en/3.0/ref/contrib/admin/#django.contrib.admin.TabularInline)